packages/ods/src/components/combobox/tests/rendering/ods-combobox.e2e.ts

```diff
@@ -185,6 +185,84 @@ describe('ods-combobox rendering', () => {
     });
   });
 
+  describe('dropdown positioning', () => {
+    it('should position the dropdown correctly on first render', async() => {
+    // Setup with a container that has limited height to force positioning
+      await setup(`
+      <div style="height: 200px; overflow: hidden; position: relative;">
+        <div style="position: absolute; bottom: 10px; width: 100%;">
+          <ods-combobox>
+            <ods-combobox-item value="item1">Item 1</ods-combobox-item>
+            <ods-combobox-item value="item2">Item 2</ods-combobox-item>
+            <ods-combobox-item value="item3">Item 3</ods-combobox-item>
+            <ods-combobox-item value="item4">Item 4</ods-combobox-item>
+            <ods-combobox-item value="item5">Item 5</ods-combobox-item>
+          </ods-combobox>
+        </div>
+      </div>
+    `);
+
+      // Focus to open dropdown
+      const inputEl = await page.find('ods-combobox >>> ods-input');
+      await inputEl.click();
+      await page.waitForChanges();
+
+      // Verify dropdown is open
+      expect(await isOpen()).toBe(true);
+
+      // Check if the dropdown is positioned at the top (since we're at the bottom of container)
+      const isPositionedTop = await page.evaluate(() => {
+        const results = document.querySelector('ods-combobox')?.shadowRoot?.querySelector('.ods-combobox__results');
+        const searchEl = document.querySelector('ods-combobox')?.shadowRoot?.querySelector('.ods-combobox__search');
+        return results?.classList.contains('ods-combobox__results--top') &&
+             searchEl?.classList.contains('ods-combobox__search--top');
+      });
+
+      expect(isPositionedTop).toBe(true);
+    });
+
+    it('should maintain correct positioning when reopening the dropdown', async() => {
+    // Similar setup as above
```

Comment:
picture each tests as independant, if test above is removed one day, this comment gets wrong